meson.build

```diff
@@ -1808,26 +1808,13 @@ xml_helper_py = find_program('tools/xml_helper.py')
 ############################################################
 
 version_tag = get_option('version-tag')
-# Check that we have either .git/ (a normal clone) or a .git file (a work-tree) and that we don't
-# get confused if a tarball is extracted in a higher-level git repository.
-if version_tag == '' and git.found() and fs.exists(project_source_root / '.git')
-        # If the working tree has no tags (CI builds), the first git-describe will fail
-        # and we fall back to project_version-commitid instead.
-        version_cmd = '''(git -C "$1" describe --abbrev=7 --dirty=^ 2>/dev/null ||
-                          echo "$2-$(git -C "$1" describe --always --abbrev=7)") |
-                         sed 's/^v//; s/-rc/~rc/' '''
-else
-        version_cmd = '''echo "$2" '''
-endif
-
 version_h = vcs_tag(
         input : 'src/version/version.h.in',
         output : 'version.h',
-        # TODO: Use 'sh' variable with meson >= 0.63.0
-        command: ['sh', '-euc', version_cmd,
-                  '_',
+        command: [project_source_root / 'tools/meson-vcs-tag.sh',
```

Comment:
Maybe, better to comment we should use `find_program()` with meson >= 0.63.0.

Reply:
I don't think it's better. The program is ever going to be used in one place only, so `find_program()` would make the output more verbose are require one more line of code. So I don't think we'd want to switch even if there was no problem with older versions.